zos.initialize-deploy.js

```diff
@@ -0,0 +1,79 @@
+const fs = require('fs');
```

Comment:
@tspoff @adibas03 Do we need to have a separate deploy script? I thought we can just use the `zos publish --network kovan` to handle that

Reply:
Not yet concluded on this,
If we are using the deploy script, it will most likely be for the post zos actions, if any can't be handled directly from cmd

Reply:
`zos publish`
`zos push`
`zos create CrowdloanFactory`
call initialize() on deployed instance w/ deployed App address from zos.<network>.json